components/ui/tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+"use client"
+
+import * as React from "react"
```

Comment:
Why are you importing all of React instead of only what you need? I also don't understand why you need the * here as well.

Reply:
That comes from shadcn... I can look into why they do it but to be honest, never even looked at this file.